doc/nrf/releases_and_maturity/releases/release-notes-2.4.0.rst

```diff
@@ -814,11 +814,11 @@ Libraries for networking
 
 * :ref:`lib_download_client` library:
 
-  * Added the :c:func:`download_client_get` function that combines the functionality of functions :c:func:`download_client_set_host`, :c:func:`download_client_start`, and :c:func:`download_client_disconnect`.
+  * Added the :c:func:`download_client_get` function that combines the functionality of functions ``download_client_set_host``, :c:func:`download_client_start`, and ``download_client_disconnect``.
```

Comment:
We normally don't edit the old release note. I could see these functions still defined in the doxygen - https://ncsdoc.z6.web.core.windows.net/PR-17481/nrf/doxygen/html/group__dl__client_ga9c6e4153ef22913ea5f57aa7aed884e7.html#ga9c6e4153ef22913ea5f57aa7aed884e7.
Hence, could be linked.
Any specific reason to update these ?

Reply:
No, it can be updated when the download client is removed.